### PR TITLE
Cross-link to additional content on dead-lettering

### PR DESCRIPTION
**Pull request by udidahan**
Replaces https://github.com/Azure/azure-docs-sdk-dotnet/pull/2283

### CODE REVIEW

File: sdk/servicebus/Azure.Messaging.ServiceBus/README.md [hunk 1]
```diff
@@ -338,6 +338,8 @@ string reason = dlqMessage.DeadLetterReason;
 string description = dlqMessage.DeadLetterErrorDescription;
 ```
 
+For more information, see the [overview of ServiceBus dead letter queues](/azure/service-bus-messaging/service-bus-dead-letter-queues).
```
Comment (jsquire) [inline sdk/servicebus/Azure.Messaging.ServiceBus/README.md:341]:
```suggestion
For more information, see the [overview of ServiceBus dead letter queues](https://learn.microsoft.com/azure/service-bus-messaging/service-bus-dead-letter-queues).
```

We'll need to make this an absolute URL, as the content will be used from both the GitHub repository and MS Learn.

**Review by jsquire — APPROVED** [review]
Thanks very much for your contribution, @udidahan!

File: sdk/servicebus/Azure.Messaging.ServiceBus/README.md (hunk 1 above)
Comment (udidahan) [reply sdk/servicebus/Azure.Messaging.ServiceBus/README.md:341]: Gotcha 👍🏼

**Comment by jsquire** [comment]
Looks good, merging things in.   Thank you, again, @udidahan.

**Comment by udidahan** [comment]
Glad this was able to go through. Thank you all!

**Pull request merged**